habitat/core/embodied_task.py

```diff
@@ -131,6 +143,50 @@ def __init__(
         self._sim = sim
         self._dataset = dataset
 
+        self._possible_actions = OrderedDict()
+        for action in config.POSSIBLE_ACTIONS:
+            action_method = registry.get_task_action(action)
+            self._possible_actions[action] = action_method
+
+    def step(
+        self,
+        action: Union[str, int],
+        episode: Type[Episode],
+        action_args: Dict[str, Any] = None,
```

Comment:
Can we add `*args, **kwargs` here and everywhere, to easily add new arguments later in future and easy backward compatibility?